### PR TITLE
Autoinst profile channels 2 #14765 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

"add-on" and "software" features in Autoyast section
https://github.com/SUSE/spacewalk/issues/14765

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
If more info is needed, we'd probably better it to https://github.com/SUSE/manager-build-profiles (readme or comments in `.xml`).

File: modules/client-configuration/pages/autoinst-profiles.adoc [hunk 1]
```diff
@@ -101,23 +98,32 @@ You can change the details and settings of your autoinstallation profiles by nav
...
 For more information about using variables in {ay} scripts, see xref:client-configuration:autoinst-profiles#variables[Variables].
 
-Here are the most important sections in the {ay} installation file for installing with SUSE Manager:
+These are the most important sections in the {ay} installation file for installing with {productname}:
 
 * ``<add-on>`` section allows to add child channels to the installation
++
+  See https://doc.opensuse.org/projects/autoyast/#Software-Selections-additional with an ``<add-on>`` example
+
+* ``<software>`` section allows to specify ``<product>`` for the Unified Installer
```
Comment (mcalmer) [inline modules/client-configuration/pages/autoinst-profiles.adoc:123]: I cannot add the comment at the right place, but "software" exists again below. I think we need it only 1 time.

**Review by mcalmer — CHANGES_REQUESTED** [review]
one duplicate info found

File: modules/client-configuration/pages/autoinst-profiles.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/autoinst-profiles.adoc:123]: thanks for spotting!

File: modules/client-configuration/pages/autoinst-profiles.adoc [hunk 2]
```diff
@@ -101,27 +98,36 @@ You can change the details and settings of your autoinstallation profiles by nav
...
 An {ay} profile consists of a [guimenu]``Label`` that identifies the profile, an [guimenu]``Autoinstall Tree`` that points to an autoinstallable distribution, various options, and, most importantly, an {ay} installation file.
 
-The {ay} installation file is an XML file that give directions to the {ay} installer. {ay} calls it a "control file."
+The {ay} installation file is an XML file that give directions to the {ay} installer.
+{ay} calls it a "control file."
 For the full syntax of {ay} installation files, see https://doc.opensuse.org/projects/autoyast/#cha-configuration-installation-options.
 
 {suse} provides templates of {ay} installation files that you can use as a starting point for your own custom files.
-You will find them at https://github.com/SUSE/manager-build-profiles in the directory named [path]``AutoYast``.
+You will find them at https://github.com/SUSE/manager-build-profiles in the [path]``AutoYast`` directory.
```
Comment (0rnela) [inline modules/client-configuration/pages/autoinst-profiles.adoc:112]:
```suggestion
You will find the templates at https://github.com/SUSE/manager-build-profiles in the [path]``AutoYast`` directory.
```

**Review by 0rnela — APPROVED** [review]

File: modules/client-configuration/pages/autoinst-profiles.adoc [hunk 3]
```diff
@@ -101,27 +98,36 @@ You can change the details and settings of your autoinstallation profiles by nav
...
+* ``<software>`` section allows to specify ``<product>`` for the Unified Installer
 
-* ``<add-on>`` section allows to add child channels to the installation
 
 * ``<general>$SNIPPET('spacewalk/sles_no_signature_checks')</general>` disables signature checks
 
-* ``<software>`` allows to specify the correct product for the Unified Installer
+* ``<software>`` allows to specify product for the Unified Installer
++
+  See https://doc.opensuse.org/projects/autoyast/#CreateProfile-Software with a "<software>" example
```
Comment (mcalmer) [inline modules/client-configuration/pages/autoinst-profiles.adoc:130]: still duplicated. I think we only need 1 `<software>` item. Now we have 2

- line 123
- line 128

**Review by mcalmer — CHANGES_REQUESTED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/client-configuration/pages/autoinst-profiles.adoc (hunk 3 above)
Comment (keichwa) [reply modules/client-configuration/pages/autoinst-profiles.adoc:127]: thanks for checking again.  something went wrong.  now done. crossed fingers...

**Pull request merged**